app.go

```diff
@@ -122,3 +122,33 @@ func (a *App) PackageRoot(pp string) {
 	a.ModelsPkg = pp + "/models"
 	a.GriftsPkg = pp + "/grifts"
 }
+
+type packageJSON struct {
+	Scripts map[string]interface{} `json:"scripts"`
+}
+
+// NodeJsScript gets the "scripts" section from package.json and
+// returns the matching script if it exists and true; nil and false
+// otherwise.
+func (a App) NodeJsScript(name string) (interface{}, bool) {
```

Comment:
NodeJs is redundant I think. NodeScript is good enough. 

I also don’t think we need the Has version as this returns a bool; though I think an error is probably better. 

We should probably read and cache that file once in the App so if we ask for multiple things it doesn’t have to keep reading the file. 

Finally is interface{} the best return value? Should it be a string, or slice of strings, or even a command?

Reply:
Ok, I pushed a new revision. I was not sure about the interface{}, since `true` worked as a value using npm, but I guess it's just because of js type conversion.